### PR TITLE
[DS-555] Prepare a Membership Calculator block to work with Layout Builder

### PR DESCRIPTION
**Pull request by aleevas**
[DS-555](https://yusa.atlassian.net/browse/DS-555)

How to check:

- [ ] Enable the LB Membership Calculator module
- [ ] create a new Landing Page with Layout Builder
- [ ] click  "+Add a new block" to the Body section
- [ ] click Create custom block
- [ ] verify that you can see a new Membership block in the list of blocks
- [ ] select this Membership block 
- [ ] save the layout
- [ ] verify that you can see fully functional Membership calculator block

### CODE REVIEW

File: modules/lb_membership_calculator/config/install/core.entity_form_display.block_content.lb_membership_calculator.default.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - block_content.type.lb_membership_calculator
+    - field.field.block_content.lb_membership_calculator.field_block
+  module:
+    - plugin
```
Comment (podarok) [inline modules/lb_membership_calculator/config/install/core.entity_form_display.block_content.lb_membership_calculator.default.yml:8]: add it to module dependencies, please https://github.com/YCloudYUSA/yusaopeny_memberships/pull/6

File: modules/lb_membership_calculator/config/optional/field.storage.block_content.field_block.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - block_content
```
Comment (podarok) [inline modules/lb_membership_calculator/config/optional/field.storage.block_content.field_block.yml:5]: add it to module dependencies, please https://github.com/YCloudYUSA/yusaopeny_memberships/pull/6

File: modules/lb_membership_calculator/config/install/core.entity_form_display.block_content.lb_membership_calculator.default.yml (hunk 1 above)
Comment (aleevas) [reply modules/lb_membership_calculator/config/install/core.entity_form_display.block_content.lb_membership_calculator.default.yml:8]: added

File: modules/lb_membership_calculator/config/optional/field.storage.block_content.field_block.yml (hunk 2 above)
Comment (aleevas) [reply modules/lb_membership_calculator/config/optional/field.storage.block_content.field_block.yml:5]: @podarok this module  is already in the dependencies list

**Pull request merged**